pkg/indexer/storer/identityUpdate.go

```diff
@@ -48,7 +50,11 @@ func NewIdentityUpdateStorer(
 }
 
 // Validate and store an identity update log event
-func (s *IdentityUpdateStorer) StoreLog(ctx context.Context, event types.Log) LogStorageError {
+func (s *IdentityUpdateStorer) StoreLog(
+	ctx context.Context,
+	event types.Log,
+	appendLog bool,
```

Comment:
Maybe we can add a comment about when you would and wouldn't want to use this flag.